frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -54,13 +54,18 @@ const InputField = styled(TextField)({
         background: "#e7e7ea",
       },
       "&:active": {
-        background:
-          "linear-gradient(0deg, rgba(255, 255, 255, 0.85), rgba(255, 255, 255, 0.85)), #0D1030;",
+        background: "#DBDBE0",
       },
     },
   },
 });
 
+// search's result options container
+const ResultGrid = styled(Grid)({
+  height: "inherit",
+  padding: "12px 16px 12px 16px",
+})
```

Comment:
didn't check locally but this might need a lint (i think maybe missing a semicolon)